p4-16/psa/PSA.mdk

```diff
@@ -489,23 +499,35 @@ copy going to the control CPU to have a software-defined header
 (defined in the P4 program) that is different than any headers on the
 packet copies going to the Egress control block.
 
+Whenever the pseudocode above indicates that a packet should be sent
+to a particular place, a PSA implementation may under some
```

Comment:
s/place/port?

Reply:
I am guessing PSA implementations will be free to drop resubmitted and recirculated packets, too, which at least right now in the PSA spec aren't described as ports.  Maybe s/to a particular place/on a particular packet path/ ?